### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by xdg2016** [eric612/MobilenetV3-Caffe#1]
Hello, your work is good, I wonder is this pretrained model trained in ImageNet dataset? did you eval it with other data? and how the model works?

**Comment by eric612** [comment]
You can see this issue :
https://github.com/eric612/MobileNet-YOLO/issues/161

Howerver , here has a problem , I can't bring the pytorch fc layer bias to caffemodel , eval in the imagenet dataset would get a poor result ,  but , I only use the backbone weights to train voc dataset. so it is fine in my works

**Comment by xdg2016** [comment]
Thank you, I want to use your trained model to finetune in my own dataset. Before doing this, I selected several images from ImageNet to eval the  accuracy of the model , however ,the model predicted all test images as label 455. I don't know why, can you help me ?
